docs/offboarding.rst

```diff
@@ -0,0 +1,162 @@
+Off-board Administrators and Journalists
+========================================
+
+When Journalists and SecureDrop Administrators leave your organization, it is important to off-board them from SecureDrop.
```

Comment:
Administrators->administrators (consistency w/ usage in docs)

Reply:
fixed in 1def8b9ee